<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>guigui</class>
 <widget class="QWidget" name="guigui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChatBot</string>
  </property>
  <widget class="QLabel" name="grad">
   <property name="geometry">
    <rect>
     <x>-40</x>
     <y>-70</y>
     <width>431</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0.959909, stop:0 rgba(255, 185, 0, 255), stop:1 rgba(37, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="voiceChat">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(47, 47, 47,200);
border: none;
border-radius:25px;
font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Voice Chat Bot</string>
   </property>
  </widget>
  <widget class="QPushButton" name="textChat">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(47, 47, 47,200);
border: none;
border-radius:25px;
font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Text Chat Bot</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Main Menu</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
